server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -132,7 +143,7 @@ public enum MetadataVersion {
      * <strong>Think carefully before you update this value. ONCE A METADATA VERSION IS PRODUCTION,
      * IT CANNOT BE CHANGED.</strong>
      */
-    public static final MetadataVersion LATEST_PRODUCTION = IBP_4_0_IV3;
+    public static final MetadataVersion LATEST_PRODUCTION = IBP_4_1_IV1;
```

Comment:
Is this change intended?

Reply:
At some point, we need to bump the LATEST_PRODUCTION for 4.1. If this PR is not going to be a suitable point for this, please let me know.